File: ex1672/go/main.go
package main

import "fmt"

func main() {
	// 2D array/slice
	accounts := [][]int{
		{1, 2, 3},
		{3, 2, 1},
	}
	maximumWealth(accounts)
}

func maximumWealth(accounts [][]int) {
	for i := 0; i < len(accounts); i++ {
		sum := 0
		for j := 0; j < len(accounts[i]); j++ {
			sum += accounts[i][j]
		}
		fmt.Printf("Sum of row %d = %d\n", i+1, sum)
	}
}

File: ex1672/java/Main.java
package ex1672.java;

public class Main {
    public static void main(String[] args) {
        int[][] array = { {1,2,3}, {3,2,1} };
        maximumWealth(array);
    }
    public static int maximumWealth(int[][] accounts) {
        for (int i = 0; i < accounts.length; i++) {
            int sum = 0;
            for (int j = 0; j < accounts[i].length; j++) {
                sum += accounts[i][j];
            }
            return sum;
        }
    }
}
